Line_6 $$ Connection con = null-->Line_9 $$ con.close()[ FD ]
Line_4 $$ DataSource ds = (DataSource) initialContext.lookup("java:jboss/datasources/" + VAULT_BLOCK_WRONG)-->Line_8 $$ con = ds.getConnection()[ FD ]
Line_2 $$ public void testRejectWrongPasswordThroughVaultDatasource() throws Exception -->Line_3 $$ Context initialContext = new InitialContext()[ CD ]
Line_4 $$ DataSource ds = (DataSource) initialContext.lookup("java:jboss/datasources/" + VAULT_BLOCK_WRONG)-->Line_5 $$ Assert.assertNotNull(ds)[ FD ]
Line_2 $$ public void testRejectWrongPasswordThroughVaultDatasource() throws Exception -->Line_5 $$ Assert.assertNotNull(ds)[ CD ]
Line_2 $$ public void testRejectWrongPasswordThroughVaultDatasource() throws Exception -->Line_4 $$ DataSource ds = (DataSource) initialContext.lookup("java:jboss/datasources/" + VAULT_BLOCK_WRONG)[ CD ]
Line_6 $$ Connection con = null-->Line_8 $$ con = ds.getConnection()[ FD ]
Line_2 $$ public void testRejectWrongPasswordThroughVaultDatasource() throws Exception -->Line_6 $$ Connection con = null[ CD ]
Line_3 $$ Context initialContext = new InitialContext()-->Line_4 $$ DataSource ds = (DataSource) initialContext.lookup("java:jboss/datasources/" + VAULT_BLOCK_WRONG)[ FD ]
Line_8 $$ con = ds.getConnection()-->Line_9 $$ con.close()[ FD ]
